assets/js/base/components/sidebar-layout/style.scss

```diff
@@ -29,28 +29,28 @@
 				margin-right: 0;
 			}
 		}
+	}
 
-		> .components-panel__body-title,
-		.components-panel__body-toggle {
-			&,
-			&:hover,
-			&:focus,
-			&:active {
-				background-color: transparent;
-				color: inherit;
-			}
+	.components-panel__body-title,
+	.components-panel__body-toggle {
+		&,
+		&:hover,
+		&:focus,
+		&:active {
+			background-color: transparent;
+			color: inherit;
 		}
+	}
 
-		.components-panel__body-toggle {
-			font-weight: normal;
-			font-size: inherit;
-			padding-left: 0;
-			padding-right: 36px;
+	.components-panel__body-toggle {
+		font-weight: normal;
+		font-size: inherit;
+		padding-left: 0;
+		padding-right: 36px;
 
-			&.components-button,
-			&.components-button:focus:not(:disabled):not([aria-disabled="true"]) {
-				color: inherit;
-			}
+		&.components-button,
+		&.components-button:focus:not(:disabled):not([aria-disabled="true"]) {
```

Comment:
not related, but this is a wild selector, good luck overriding it 😆

Reply:
Agree, but this selector is indeed to override a selector from Gutenberg. :sweat: